api/env.js

```diff
@@ -1,8 +1,15 @@
+const crypto = require('crypto');
 const dotenv = require('dotenv');
 
 const defaults = {
-  PORT: 8000
+  PORT: 8000,
+  SESSION_SECRET: crypto.randomBytes(32).toString('hex')
 };
 
 dotenv.config();
-Object.assign(process.env, defaults);
+
+Object.keys(defaults).forEach((env) => {
```

Comment:
so we're adding new fields from `defaults` obj but not overriding anything already in `process.env`. is that equivalent to: `process.env = Object.assign({}, defaults, process.env)` ? (though perhaps it's bad practice to reassign `process.env`)

Reply:
Oh, huh!  I didn't know `Object.assign` could take more than two arguments.  I'll update, I prefer your way.  😄

Reply:
👍